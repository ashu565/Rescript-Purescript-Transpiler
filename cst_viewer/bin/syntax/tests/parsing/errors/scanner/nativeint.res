let nativeint = 3n
