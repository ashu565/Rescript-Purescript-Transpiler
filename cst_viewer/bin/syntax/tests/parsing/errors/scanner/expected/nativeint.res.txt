
  Syntax error!
  tests/parsing/errors/scanner/nativeint.res:1:18

  1 │ let nativeint = 3n
  2 │ 

  Unsupported number type (nativeint). Did you mean `3`?

let nativeint = 3n